lua/class_definitions.lua

```diff
@@ -84,11 +84,6 @@ classes = {
             nv_cached = { type = "bool", writable = true },
         },
         functions = {
-            { name = "add_effect", rval = nil, args = { "string", "int" } },
```

Comment:
My mistake, there was a version on the forum that uses add_effect. http://smf.cataclysmdda.com/index.php?topic=11353.0